crates/simulation/src/main.rs

```diff
@@ -449,7 +448,7 @@ async fn create_window_surface<'w>(gpu: &Gpu, window: winit::window::Window) ->
         width,
         height,
         present_mode: wgpu::PresentMode::Fifo,
-        alpha_mode: wgpu::CompositeAlphaMode::PreMultiplied,
+        alpha_mode: wgpu::CompositeAlphaMode::Opaque,
```

Comment:
This change in isolation seems to work fine on my end, and I can't remember any reason to prefer `PreMultiplied` here.